cmd/sys-mgmt-agent/res/docker/configuration.toml

```diff
@@ -0,0 +1,64 @@
+# This TOML file now incorporates a now-defunct manifest file, which had specified the services managed by the SMA.
+# The SMA needs the details provided in this TOML to stop/start/restart each service that the SMA manages.
+# What about specifying the edgexsecurity service to use?
+# Different versions of this manifest may exist depending on how/where EdgeX is deployed (Docker v. Snappy, Windows v. Linux, etc.).
+# In the first release of the SMA, the manifest will be static.
+# In the future, the manifest may be more dynamic or even provided by some 3rd party orchestrator.
+
+ApplicationName = 'edgex-sys-mgmt-agent'
+ConsulProfilesActive = 'go'
+ValidateCheck = false
+AppOpenMsg = 'This is the System Management Agent Service'
+FormatSpecifier = '%(\\d+\\$)?([-#+ 0(\\<]*)?(\\d+)?(\\.\\d+)?([tT])?([a-zA-Z%])'
+ServicePort = 48090
+ServiceTimeout = 5000
+CheckInterval = '10s'
+
+[Logging]
+EnableRemote = false
+File = '/edgex/logs/edgex-sys-mgmt-agent.log'
+
+# OperationsType = 'os'
+OperationsType = 'docker'
+ComposeUrl = 'https://raw.githubusercontent.com/edgexfoundry/developer-scripts/master/compose-files/docker-compose-delhi-0.7.0.yml'
```

Comment:
Shouldn't this be deleted too since now the ComposeUrl will be living inside the executable that we run instead of in the SMA?